pkgconfig/mbedcrypto.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix="@CMAKE_INSTALL_PREFIX@"
+includedir=@PKGCONFIG_INCLUDEDIR@
+libdir=@PKGCONFIG_LIBDIR@
+
+Name: @PROJECT_NAME@
+Description: @CMAKE_PROJECT_DESCRIPTION@
+URL: @CMAKE_PROJECT_HOMEPAGE_URL@
```

Comment:
You probably meant [PROJECT_DESCRIPTION](https://cmake.org/cmake/help/latest/variable/PROJECT_DESCRIPTION.html) and [PROJECT_HOMEPAGE_URL](https://cmake.org/cmake/help/latest/variable/PROJECT_HOMEPAGE_URL.html) - the `CMAKE_` variants don't work as expected when mbedtls is used as a subproject.

Reply:
Sorry I have very little experience with cmake, I use autotools mostly. What about the usage of `CMAKE_INSTALL_PREFIX` and in `mbedtls/CMakeLists.txt` the usages of `CMAKE_INSTALL_INCLUDEDIR` and  `CMAKE_INSTALL_LIBDIR`? Also, if it's a subproject, my initial reaction is that it is being built into something else, so are the pc files even relevant?

Reply:
> What about the usage of `CMAKE_INSTALL_PREFIX` and in `mbedtls/CMakeLists.txt` the usages of `CMAKE_INSTALL_INCLUDEDIR` and `CMAKE_INSTALL_LIBDIR`?

I only meant the `CMAKE_` variants of `PROJECT_DESCRIPTION` and the other variables generated by the `project()` call, not all variables which start with `CMAKE_` - the others are fine :)

> Also, if it's a subproject, my initial reaction is that it is being built into something else, so are the pc files even relevant?

Yeah, it probably doesn't matter, but there's still no reason to use those top-level variables when you don't need them.